spec/models/repository/git_spec.rb

```diff
@@ -63,7 +63,7 @@
     context 'with managed config' do
       let(:config) { { manages: managed_path } }
       let(:project) { FactoryGirl.build :project }
-      let(:identifier) { "#{project.identifier}.git" }
+      let(:identifier) { project.identifier + '.git' }
```

Comment:
Yes, it is [just a convention to use `.git`](http://git.661346.n2.nabble.com/importance-of-git-extension-for-bare-repos-td7289632.html) to denote bare repositories.